cfme/tests/control/test_bugs.py

```diff
@@ -352,3 +402,59 @@ def test_alert_ram_reconfigured(hardware_reconfigured_alert):
     view = navigate_to(hardware_reconfigured_alert, "Details")
     attr = view.hardware_reconfigured_parameters.get_text_of("Hardware Attribute")
     assert attr == "RAM Increased"
+
+
+@pytest.mark.tier(2)
+@test_requirements.alert
+def test_alert_for_disk_usage(setup_disk_usage_alert):
+        """
+        Bugzillas:
+            * 1658670, 1672698
+        Polarion:
+            assignee: jdupuy
+            casecomponent: Control
+            caseimportance: medium
+            initialEstimate: 1/6hr
+            testSteps:
+                1. Go to Control > Explorer > Alerts
+                2. Configuration > Add new alert
+                3. Based on = Server
+                4. What to evaluate = Expression (Custom)
+                5. Driving Event = "Appliance Operation: Server High /var/www/miq/vmdb/log Disk Usage"  # noqa
```

Comment:
noqa here?

Reply:
@jawatts this line is over 100 columns and I wasn't sure if it it would mess up the polarion metadata if I moved it across two lines. Do you know?

Reply:
@john-dupuy I would just format it like everything else here, split it on a new line with the correct indentations

Reply:
gotcha, I will do that